build.xml

```diff
@@ -133,6 +133,20 @@
     <mkdir dir="build"/>
   </target>
 
+  <property name="ivy.install.version" value="2.3.0" />
+  <condition property="ivy.home" value="${env.IVY_HOME}">
```

Comment:
Right, that's the plan. (Of course, running `download-ivy` is always done manually and is optional.)